release-content/0.14/migration-guides/_guides.toml

```
@@ -515,10 +515,10 @@ areas = ["Rendering"]
 file_name = "12559_Remove_needless_color_specializaion_for_SpritePipeline.md"
 
 [[guides]]
-title = "Improve performance by binning together opaque items instead of sorting them."
+title = "Sorted and Binned Render Phase Items, Resources, and non-meshes"
```

```suggestion
title = "Sorted and binned render phase items, resources, and non-meshes"
```

I personally don't capitalize non-proper-nouns in these titles, so I'd rather do the same here for consistency.